ci/cron/src/Main.hs

```diff
@@ -150,19 +159,20 @@ build_docs_folder path versions latest = do
                 then do
                     putStrLn "  Checks, reusing existing."
                     copy (old </> version) $ new </> version
-                    return $ Just version
+                    return $ Just gh_version
                 else do
                     putStrLn "  Check failed. Rebuilding..."
                     build version new
-                    return $ Just version
+                    return $ Just gh_version
             else do
                 putStrLn "  Not found. Building..."
                 build version new
-                return $ Just version)
+                return $ Just gh_version)
         putStrLn $ "Copying latest (" <> latest <> ") to top-level..."
         copy (new </> latest </> "*") (new <> "/")
         putStrLn "Creating versions.json..."
-        create_versions_json documented_versions new
+        create_versions_json (filter (not . prerelease) documented_versions) (new </> "versions.json")
```

Comment:
I think this would be clearer if you first split the versions via `partition` instead of filtering twice once with the negated condition (and it’s faster but that’s hardly relevant here).

Reply:
Did not know about the partition list, will amend.